src/config/defaults.ts

```diff
@@ -5,6 +5,9 @@ export const defaults: Config = {
   serverURL: '',
   defaultDepth: 2,
   maxDepth: 10,
+  defaultMaxTextLength: 500,
```

Comment:
I think this should be quite a bit bigger than 500 by default. 500 is quite small and would produce breaking changes for many of our users. I'd vote for a _**significantly**_ larger value here as the purpose of the PR is to prevent against malicious activity, and a 500 character text string is quite small as it is. I'd say like, way way higher. What do you think?

Reply:
Will increase to 40k